packages/caliper-ethereum/test/connectorFactory.js

```diff
@@ -0,0 +1,51 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const sinon = require('sinon');
+const expect = chai.expect;
+const ConfigUtil = require('@hyperledger/caliper-core').ConfigUtil;
+const path = require('path');
+
+const { ConnectorFactory } = require('../lib/connectorFactory');
+const EthereumConnector = require('../lib/ethereum-connector');
+
+describe('ConnectorFactory', function() {
+    let ethereumConnectorStub;
+    let tempConfigFilePath;
+
+    beforeEach(() => {
+        ethereumConnectorStub = sinon.stub(EthereumConnector, 'constructor').returns({});
```

Comment:
This line isn't required as it isn't possible to stub the creation of a class and there is no issue with letting the connectorFactory create an instance of EthereumConnector